packages/client3/tailwind.config.js

```diff
@@ -38,6 +38,7 @@ module.exports = {
       },
       transparent: "transparent",
       current: "currentColor",
+      white: "#FFFFFF",
```

Comment:
This is fine. On the weekend I was playing around with a project and you could import `tailwindcss` default colors:
import at the top:
`const colors = require('tailwindcss/colors');`

then set your white as:
`white: colors.white`

I thought I'd share since we're all still learning. 

Ref: https://tailwindcss.com/docs/customizing-colors#using-the-default-colors

Reply:
Ahh..that's cool..

Reply:
Done